recipes/pixstem/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pixstem" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 400da42fb11115b62548e8227f26b97c573870a72ebb5dbe66af437bec47c4a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - scipy
+    - numpy >=1.13
+    - h5py
+    - matplotlib >=2.0
+    - dask
+    - tqdm
+    - scikit-image >=0.13
+    - scikit-learn
+    - hyperspy >=1.4
+
+test:
+  imports:
+    - pixstem
+  commands:
+    - export MPLBACKEND=agg  # [unix]
+    - set MPLBACKEND=agg  # [win]
+    - python -c "import pixstem.api"
+
+about:
+  home: https://pixstem.org
+  license: GPL-3.0
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

The `RECIPE_DIR` part is no longer needed as conda will automatically look in there if it isn't found in the build directory:
```suggestion
  license_file: LICENSE
```

---

ok, good to know.

I got the line from https://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually , which I guess might be out-of-date.

---

It seems `readthedocs` is out of date with the docs that are [linked from the homepage](http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=packaging%20license#packaging-the-licence-manually) as that was fixed a few days okay. I've opened an issue about it (https://github.com/conda-forge/conda-forge.github.io/issues/735), thanks for letting me know! 😄

---

Nice! The reason I was using the `rtd` page was because that is the top hit when googling "conda forge meta.yaml", while the conda-forge.org page is on page 2.